src/Scripting/Core/Hosting/AssemblyLoader/ShadowCopy.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.CodeAnalysis.Scripting.Hosting
     /// <summary>
     /// Represents a shadow copy of a single file.
     /// </summary>
-    public sealed class ShadowCopy
+    public sealed class FileShadowCopy
```

Comment:
We have MetadataShadowCopy, so FileShadowCopy is alike. It's a shadow copy of a file thus FileShadowCopy.